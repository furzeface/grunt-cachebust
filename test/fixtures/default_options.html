<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Index</title>
  <link rel="stylesheet" href="styles/main.css" />
  <script src="scripts/main.js"></script>
</head>
<body>
  <script src="scripts/moar.js"></script>
</body>
</html>
