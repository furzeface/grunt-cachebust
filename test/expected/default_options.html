<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Index</title>
  <link rel="stylesheet" href="styles/main.css?hash=88fe72e5a58f58fc41e6f232413a7195" />
  <link rel="stylesheet" href="http://cdn.com/styles/main.css" />
  <script src="scripts/main.js?hash=f4926182610663646524e1a88f9f3622"></script>
</head>
<body>
  <script src="scripts/moar.js?hash=ec153275ca18bfd6eb634ef12ee6c02b"></script>
  <script src="http://best-cdn-ever.com/scripts/evenMoar.js"></script>
</body>
</html>
